tests/assets/workflow_example/daylightfactor.yaml

```diff
@@ -167,7 +167,7 @@ flow:
       arguments:
         parameters:
           - name: scene-file-paths
-            value: "{{ workflow.inputs.parameters.scene-file-paths }}"
+            value: "{{ workflow.inputs.parameters.scene-file-paths.value }}"
```

Comment:
Interesting! What are the other properties here that one can use?

Reply:
At this stage the function fetching values recurses through the entire object looking for `{{ workflow.x.y...z }}` so pretty much anything in the workflow object.